DllImportGenerator/DllImportGenerator/DllImportGeneratorOptions.cs

```diff
@@ -1,12 +1,15 @@
-using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.Diagnostics;
-using System;
-using System.Collections.Generic;
-using System.Diagnostics;
-using System.Runtime.InteropServices;
 
 namespace Microsoft.Interop
 {
+    record DllImportGeneratorOptions(bool GenerateForwarders, bool UseMarshalType) : InteropGenerationOptions(UseMarshalType)
```

Comment:
Public?

Reply:
This record is specific to the options for the GeneratedDllImport generator, so I don't think we need to expose it publically.